db/schema.rb

```diff
@@ -340,6 +340,8 @@
     t.integer "average_lifetime"
     t.float "average_score"
     t.float "average_apy"
+    t.float "withdrawal_fee"
+    t.float "deposit_fee"
```

Comment:
Are these two fields added by you?

Reply:
There're from different task, merged to master